lib/fetch/body.js

```diff
@@ -253,7 +251,7 @@ function safelyExtractBody (object, keepalive = false) {
 }
 
 function cloneBody (body) {
-  // To clone a body body, run these steps:
+  // To clone a body, run these steps:
```

Comment:
this comment should be reverted, see https://fetch.spec.whatwg.org/#concept-body-clone

Reply:
can you re-run the tests? i just want to make sure the failure is not related to this pull request

Reply:
Done. reverted the change.